apps/web/app/features/stacking/start-pooled-stacking/utils/utils-delegate-stx.ts

```diff
@@ -111,6 +112,14 @@ export function createDelegateStxMutationOptions({
 
       const delegateStxOptions = getDelegateStxOptions(values, poxInfo, network);
 
+      await analytics.track('pooled_stacking_started', {
```

Comment:
Let's add this sort of contract-parameter tracking on the extension side, too, for thoroughness (and without tracking the executor's address per se, for PII-protection purposes). That way, we can see the range of contracts, functions and parameters being executed in general via the extension in addition to seeing them triggered here on the web side.